### ISSUE TITLE
CI checks

### ISSUE THREAD

**Issue by mike-allaway** [bear-rsg/social-bodies#13]
- Check that all necessary CI are in place, e.g. accessibility, flake8, Django unit tests.
- Make necessary changes to code as a result of failing any CI, e.g. fix any accessibility issues that pa11y raises
